.application-version

```diff
@@ -0,0 +1 @@
+e116268968fec6ef4114cf6a5e5eb9bf00d98823
```

Comment:
should this file be here? Perhaps it is useful to have one that contains 'not set' or similar?